.github/workflows/dry-run-test.yaml

```diff
@@ -19,7 +19,7 @@ jobs:
           go-version: '1.16'
       - name: Install kubectl-storageos
         run: |
-          make _build-pre
+          make _build
```

Comment:
As i know if we pass a concrete version of ondat, plugin skips version validation. Just FYI, we don't have to change code here.